### ISSUE TITLE
Unable to drag and drop in Google Chrome (dynamic-rows or grid)

### ISSUE THREAD

**Issue by navarr** [magento/magento2#8495]
### Preconditions

1. Magento 2.1.3
2. Chrome 56.0.2924.87 (Official Build) (32-bit)
3. Windows 10 (x64)

### Steps to reproduce

1. Visit the Sales Order Grid
2. Attempt to rearrange columns by clicking one, holding down the mouse button, and moving the mouse

### Expected result

1. Columns move with the mouse

### Actual result

1. Nothing happens

This seems to be due to the dnd.js attempting to determine `isTouchDevice` and believing that Chrome is a touch device.

The code in question `typeof document.ontouchstart !== 'undefined'` returns "true" on my instance of Chrome.  (None of my screens are touch screens, and even if they were I would still have a mouse and want to use it).

Code is located at: [(a)](https://github.com/magento/magento2/blob/develop/app/code/Magento/Ui/view/base/web/js/grid/dnd.js#L14), [(b)](https://github.com/magento/magento2/blob/develop/app/code/Magento/Ui/view/base/web/js/dynamic-rows/dnd.js#L14)

In a local environment we have overcome this issue by overriding the JavaScript files via the Magento 2 requirejs loading system and putting in our own versions of the files that, instead of listening to either mouse or touch - listens to both.  This has not, however, been extensively tested.

**Comment by navarr** [comment]
While I have not tested this on Magento 2.1.4, the fact that the JavaScript files in question have not been modified leads me to believe the issue will still be present in that version.

**Comment by vityakopin** [comment]
Hi @navarr ! Thanks for reporting this issue. Unfortunately, I could not reproduce the issue as you described it. I try to do it on 2.1.3 and 2.1.4 on Google Chrome Version 59.0.3071.115 (Official Build) (64-bit) launched on Windows 10 64-bit. Please provide the detailed steps we must follow to reproduce this issue. In addition, identify the web server you are running, the versions of PHP and MySQL, and any other information needed to reproduce your issue.

**Comment by navarr** [comment]
A lot has changed in my local environment since I initially opened this ticket - for example, my laptop now has a touch screen.

I've tried this again in 2.1.7, Win 10 64-bit, Chrome v59.0.3071.115 (Official Build) (64-bit).

I still cannot drag columns in grids as I believe I should be able to do.  However, an easier catch is trying to drag "Custom Options" in a Product.

https://gfycat.com/DefensiveGloomyCats

In this video I start by trying to drag the columns using the handle with my house, but they do not.

I then try to drag with my fingers - to disastrous effect (though that is unrelated to this ticket).

I'm using PHP 7.0 and MySQL 5.7.

**Comment by vityakopin** [comment]
Thanks for reporting this issue. I could reproduce the issue only on device with touch screen. Internal ticket MAGETWO-70756 was created.

**Comment by andrewhowdencom** [comment]
Also reproducible on Firefox, at least in my build. 

Machine: Dell XPS 15 with Touch Screen support
Browser: Google Chrome | 60.0.3112.50 (Official Build) beta (64-bit) || Firefox 54.0 (64-bit)
OS: Debian testing (~9), with Gnome / X Display manager

Suuper dirty workaround: 
1. visit chrome://flags
2. Find "Touch Events API Mac, Windows, Linux, Chrome OS"
3. Disable it
4. Reboot chrome

A quick test shows this to then work correctly.

**Comment by navarr** [comment]
This ticket is labeled "needs update" - is it waiting on something external?

**Comment by magento-engcom-team** [comment]
@navarr, thank you for your report.     
    
Unfortunately, we are archiving this ticket now as it did not get much attention from both Magento Community and Core developers for an extended period. This is done in an effort to create a quality, community-driven backlog which will allow us to allocate the required attention more easily.

Please feel free to comment, reopen or create new ticket according to the [Issue reporting guidelines](https://github.com/magento/magento2/wiki/Issue-reporting-guidelines) 
if you are still facing this issue on the latest `2.4-develop` branch. Thank you for collaboration.

**Comment by magento-engcom-team** [comment]
Hi @navarr. 

Thank you for your report and collaboration!

The related internal Jira ticket `MC-24739` was closed as `non-reproducible` in `2.4-develop`.
It means that Magento team either unable to reproduce this issue using provided _Steps to Reproduce_ from the **_Description section_** on clean Magento instance or the issue has been already fixed in the scope of other tasks.

But if you still run into this problem please update or provide additional information/steps/preconditions in the **_Description section_** and reopen this issue.
